src/metabase/driver/hive_like.clj

```diff
@@ -36,7 +52,7 @@
 
 (defmethod sql-jdbc.sync/database-type->base-type :hive-like
   [_ database-type]
-  (condp re-matches (u/lower-case-en (name database-type))
```

Comment:
I see this line gets changed which I introduced a few months back because our databricks runtime/hive spir out capital types e.g INT instead of int. 

I wrote a test as well and I see the test has not been changed so I assume the test is now failing. Did I update the readme about running the tests?

I'd highly appreciated if you keep the lowercasing in, some of us will need it. 

This is something that should preferably be fixed in the metabase majntained sparj driver. I already supplied a similar PR there which was approved but never merged i believe. 

Unfortunately Im heading different directions and won't be using databrjcks in my day job anymore. So also not actively contributing anytime soon

Reply:
Its 5:50am and was laying in bed browsing my GH app when I noticed this line and got triggered. 

I should have started with thank you for your efforts! 😇🙏 its great to notice others adopting and contributing!

Its hard to review this on my phone. I'll see if i can do a proper review on my laptop today.

Reply:
I can confirm rollbacking that change makes that works for us.

Reply:
FYI driver compiled but with `EnableArrow=0;` in the JDBC parameter string